packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -474,6 +539,12 @@ function isArbitrumChain(
   return typeof (chain as ArbitrumNetwork).parentChainId !== 'undefined'
 }
 
+function isBaseChain(
```

Comment:
Why do we need `isBase` property if we can get it from `isNetwork`? now instead we use them interchangeably